<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registroDialog</class>
 <widget class="QWidget" name="registroDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuevo Usuario</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 255);</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="nuevoNLabel">
     <property name="text">
      <string>Nombre:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="nuevoNLE"/>
   </item>
   <item row="3" column="1">
    <widget class="QDateEdit" name="nuevoFDE">
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="nuevoULabel">
     <property name="text">
      <string>Nombre de Usuario:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="nuevoULE"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="nuevoCLabel">
     <property name="text">
      <string>Contraseña:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="nuevoCLE">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="nuevoMLabel">
     <property name="text">
      <string>Correo Electrónico:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="nuevoMLE"/>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="registrarPB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>Registrarse</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="nuevoFLabel">
     <property name="text">
      <string>Fecha de Nacimiento:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="text">
      <string>Nuevo Usuario</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="registroLabel">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bukin</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
